run-tests.sh

```diff
@@ -135,8 +176,23 @@ EOM
 echo 'INFO: installing dependencies'
 dryrun npm ci
 
-exitCode=0
+################################ Disable Test Tags #####################################
+
+runTestsIfServiceVersion "@multipartUpload" "fence" "2.8.0"
+runTestsIfServiceVersion "@centralizedAuth" "fence" "3.0.0"
 
+# environments that use DCF features
+# we only run Google Data Access tests for cdis-manifest PRs to these
+envsRequireGoogle="dcp.bionimbus.org gen3.datastage.io nci-crdc-staging.datacommons.io nci-crdc.datacommons.io"
+
+#
+# DataClientCLI tests require a fix to avoid parallel test runs
+# contending over config files in the home directory
+#
+doNotRunRegex="@dataClientCLI"
```

Comment:
```suggestion
donot '@dataClientCLI'
```
`doNotRunRegex` should be initialized earlier